tests/acceptance/ConfirmTitleAlignmentSettingsInALCBlockCest.php

```diff
@@ -48,7 +48,7 @@ function createWelcomeNewsletter(\AcceptanceTester $I) {
     $I->waitForText($post_title, 60);
 
     // assert no vertical element with two text block is present
-    $I->cantSeeElement('.mailpoet_container_vertical > .mailpoet_text_block + .mailpoet_text_block');
+    $I->cantSeeElementInDOM('.mailpoet_container_vertical > .mailpoet_text_block + .mailpoet_text_block');
```

Comment:
This was meant to be added to the previous commit or does it play some role in this one?

Reply:
Actually, what this comment does? It seems to rename a test and change `automatedLatestContent` to `automatedLatestContentLayout` but why? What is the difference between those two and why we have both?

Reply:
We only need  `automatedLatestContentLayout` and that is the actual version. We only keep `automatedLatestContent` for compatibility reasons. See this conversation: https://mailpoet.slack.com/archives/G6JE6CDBK/p1550570233025200